homework/index.js

```diff
@@ -29,12 +31,115 @@
     return elem;
   }
 
-  function renderRepoDetails(repo, ul) {
-    createAndAppend('li', ul, { text: repo.name });
+  function addCont(login, img, contributeNumber, parent, link) {
+    const tr = createAndAppend('tr', parent);
+    const imageColumn = createAndAppend('th', tr);
+    createAndAppend('img', imageColumn, {
+      src: img,
+      class: 'contributorImg',
+    });
+    const name = createAndAppend('td', tr);
+    createAndAppend('a', name, {
+      href: link,
+      target: '_blank',
+      text: login,
+    });
+    const contributeNamePlace = createAndAppend('td', tr);
+    createAndAppend('span', contributeNamePlace, {
+      text: contributeNumber,
+      class: 'contributeNumber',
+    });
+  }
+
+  function addRow(property, key, parent) {
+    const tr = createAndAppend('tr', parent);
+    createAndAppend('th', tr, {
+      text: `${property} :`,
+    });
+    createAndAppend('td', tr, { text: key });
+    return tr;
+  }
+
+  let optionNumber = 0;
+  function madeOption(repo, selectButton) {
+    createAndAppend('option', selectButton, {
+      text: repo.name,
+      value: optionNumber,
+    });
+    optionNumber++;
+  }
+
+  function renderContributorDetails(conts, contributorsContainer) {
+    const table = createAndAppend('table', contributorsContainer);
```

Comment:
An HTML table is not the best way to represent a list of contributors. Please take into account that your application (this one, or a future one) may need to be used by users that are visually impaired and will require on a screen reader to use your application. Therefore, for a list of contributors you should use an HTML element that semantically represents a list: in this case an unordered would be the best choice.

Reply:
Please follow up on this point, or indicate why not. See also message in the class23 channel with respect to ARIA-compliance.